src/app/package-json.js

```diff
@@ -168,13 +168,6 @@ const packageJSON = (current, context) => {
     'author': context.author,
     'license': context.licenseName,
 
-    'browserify': {
-      transform: [
-        'browserify-shim',
-        'browserify-versionify'
```

Comment:
I don't think ordering should matter in this case, but it's something to consider. I'll change this back to do versionify always and shim only for dist.